package.json

```diff
@@ -70,7 +71,9 @@
     "lint": "eslint --max-warnings 0 --ext js . && eslint --max-warnings 0 --ext ts .",
     "build": "node scripts/build",
     "watch": "node scripts/build --watch",
+    "coverage": "ASC_FEATURES=\"*\" c8 npm run test:coverage",
     "test": "npm run test:parser && npm run test:compiler -- --parallel && npm run test:browser && npm run test:asconfig && npm run test:transform",
+    "test:coverage": "npm run test:parser && npm run test:compiler && npm run test:browser && npm run test:asconfig && npm run test:transform",
```

Comment:
Looks like the `test:coverage` step can be included in `coverage` as it isn't very useful on its own. Even better perhaps, if the `test` step could detect that it runs under `c8` and then disables parallel execution - so the `test` step can be reused?

Reply:
Yes, fixed now

Reply:
Hi Dcode,
Considering the compatibility of Windows and POSIX system. I prefer the `process.argv`. currently the issue is, `c8` need a something like `zygote process` process, so it cannot use `&&` on its command line.
So I create `coverage` and `test:coverage` tasks, the `test:coverage` will have `--coverage` option for specific test `test:compiler`
The other option is we can create a `test.js` file and use it to control the environment variables and argument variables, with this option, we only need one `coverage` task.
What do you think?
Regards

Reply:
Toying around with c8 a little, I found that instead of having a separate `--coverage` option, we can detect that `c8` is being used by checking `process.env.NODE_V8_COVERAGE != null` and then turn off parallel testing. Looks like this allows to reuse the `test` step?

Reply:
@dcodeIO 
yes, we can use this option, I finished the modification